src/ray/stats/metric_exporter.cc

```diff
@@ -115,6 +135,22 @@ OpenCensusProtoExporter::OpenCensusProtoExporter(const int port,
   client_.reset(new rpc::MetricsAgentClient(address, port, client_call_manager_));
 };
 
+/// Hack. We want to add GlobalTags to all our metrics, but gRPC OpenCencus plugin is not
+/// configurable at all so we don't have chance to add our own tags. We use this hack to
+/// append the tags in export time.
+void OpenCensusProtoExporter::addGlobalTagsToGrpcMetric(
```

Comment:
Is it unit testable in metrics_exporter_test.cc?

Reply:
yes but let's make it a TODO

Reply:
Hmm why? When will you do it?

Reply:
added cc test. it's a bit special (has "global" settings) so I put it into a separate cc test file.